tuf/scripts/repo.py

```diff
@@ -295,6 +295,15 @@ def delegate(parsed_arguments):
 
     repository.targets.load_signing_key(targets_private)
 
+    # Generate the delegate targets file
+    for key in parsed_arguments.pubkeys:
+      delegate_private = import_privatekey_from_file(
+        os.path.join(parsed_arguments.path, key[:-4]),
+        parsed_arguments.pw,
+      )
+
+      repository.targets(parsed_arguments.delegatee).load_signing_key(delegate_private)
```

Comment:
From what I can see we are not generating a new file in this code block.
I know we are generating a new delegate targets file later on and I am not sure if this comment
helps or confuses where it's placed.

Reply:
Yeah true, indeed we load the key so that the delegate targets role can be generated, I'm updating it